data/kaohsiung/second-space.md

```diff
@@ -0,0 +1,53 @@
+---
+redirect_old_city: taiwan
+added: 2025-11-01
+name: Second Space Coworking
+type: Coworking Space
+area: Kaohsiung
+google_maps: https://maps.app.goo.gl/UkrK4NgSybjPVJCG6
+coordinates: 22.626005,120.2989725
+address:
+  en: 12-5, Zhongshan 1st Rd, Sinsing District, Kaohsiung City, 800
+  zh-tw: 800高雄市新興區中山一路12之5號
+station: Central Park
+hours: 0:00 - 24:00
+wifi: 5
+speed: 500
+power: 4
+vacancy: 3
+comfort: 4
+quiet: 4
+food: 2
+drinks: 3
+price: 5
```

Comment:
`price: 5` would really only be reserved for spaces that are completely free to use? It looks like you need a day pass or monthly membership to use the space?

Are snacks and drinks included in the fee? If so, maybe `price: 4` would be more suitable.

Reply:
Yeah, makes sense! We do a free trial day but then it's ~US$30 per month (or a day pass)

 I will update the PR 👋👍